RecoTauTag/RecoTau/plugins/PFRecoTauLostTrackPlugin.cc

```diff
@@ -0,0 +1,83 @@
+/*
+ * =============================================================================
+ *       Filename:  PFRecoTauLostTrackPlugin.cc
+ *
+ *    Description: Add references to tracks of tau-charged-hadrons built on 
+ *                 top of a track
+ *
+ *        Created:  25/04/2022
+ *
+ *         Authors:  Michal Bluj (NCBJ, Warsaw)
+ *
+ * =============================================================================
+ */
+
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+
+#include "RecoTauTag/RecoTau/interface/RecoTauBuilderPlugins.h"
+#include "DataFormats/TauReco/interface/PFTau.h"
+#include "DataFormats/TauReco/interface/PFRecoTauChargedHadron.h"
+#include "DataFormats/TauReco/interface/PFRecoTauChargedHadronFwd.h"
+
+namespace reco {
+  namespace tau {
+
+    class PFRecoTauLostTrackPlugin : public RecoTauModifierPlugin {
+    public:
+      explicit PFRecoTauLostTrackPlugin(const edm::ParameterSet&, edm::ConsumesCollector&& iC);
+      ~PFRecoTauLostTrackPlugin() override;
+      void operator()(PFTau&) const override;
+      void beginEvent() override;
+      void endEvent() override;
+
+    private:
+      edm::Handle<reco::TrackCollection> tracks_;
+      edm::EDGetTokenT<reco::TrackCollection> track_token_;
+      int verbosity_;
```

Comment:
```suggestion
      const edm::EDGetTokenT<reco::TrackCollection> track_token_;
      const int verbosity_;
```
and initialize them in the class initialization list